plugins/dialogui/plugin.js

```diff
@@ -1294,7 +1334,13 @@ CKEDITOR.plugins.add( 'dialogui', {
 				}
 				children[ 0 ].getElement().focus();
 			},
-
+			click: function() {
+				var radioGroup = this.selectParentTab();
+				// Update currentFocusIndex after click on the given radio element. Otherwise, click
+				// will move the focus but the focus index will not be updated, causing moving the
+				// focus incorrectly based on the previous focus index while use Tab or Shift + Tab key (#439).
```

Comment:
```suggestion
				// Update currentFocusIndex after clicking on the given radio element. Otherwise, click
				// will move the focus but the focus index will not be updated, causing moving the
				// focus incorrectly based on the previous focus index while using Tab or Shift + Tab key (#439).
```